package.json

```diff
@@ -12,9 +12,11 @@
     "customize-cra": "^0.8.0",
     "firebase": "^7.14.2",
     "formik": "^2.0.3",
+    "husky": "^4.2.5",
     "konami-code": "^0.2.1",
     "less": "^3.9.0",
     "less-loader": "^5.0.0",
+    "lint-staged": "^10.2.2",
```

Comment:
What are the benefits of using lint-staged versus doing it this way?
![image](https://user-images.githubusercontent.com/8893880/81467195-ecf23780-918b-11ea-8e2c-31b6c70e0088.png)

Reply:
I was using it as a way to only check staged files rather than enforcing everything. This would be helpful if we decided to do something like a phased roll-out of Airbnb styling so that only newly changed files need to be fixed and not the entire codebase. I'll switch to the way you provided since in this case, we already know the rest of the code would pass linting.